core/src/modules/ops/destination.ts

```diff
@@ -926,23 +926,40 @@ export namespace DestinationOps {
     const locks: Awaited<ReturnType<typeof getLock>>[] = [];
 
     try {
+      const exportIds = await Export.sequelize
+        .query<[[{ id: string }]]>(
+          {
+            query: `
+        SELECT
+          "id"
+        FROM (
+          SELECT
+            "id",
+            ROW_NUMBER() OVER (PARTITION BY "exports"."recordId" ORDER BY "exports"."createdAt" DESC) AS __rownum
+          FROM
+            "exports"
+          WHERE
+            "exports"."destinationId" = ?
+            AND "exports"."recordId" IN (?)) AS __ranked
+        WHERE
+          "__ranked"."__rownum" = 1;`,
+            values: [
+              destination.id,
+              givenExports.map(({ recordId }) => recordId),
+            ],
+          },
+          undefined
+        )
+        .then((exports) => exports[0].map((e) => e.id));
```

Comment:
I think you can skip this `[0]` if you add this kind of thing -> https://github.com/grouparoo/grouparoo/blob/main/core/src/modules/ops/record.ts#L1062-L1065 to get an object per row.

I think what you are getting back now is something like  rows -> exports -> id

Reply:
Oh, I like this!